src/style.css

```
@@ -0,0 +1,123 @@
+html,
+body,
+form,
+img,
+div {
+  margin: 0;
+}
+
+@font-face {
+  font-family: "AvenirBook";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "AvenirBold";
+  font-style: normal;
+  font-weight: 400;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.search--bars--container {
+  margin: 20px 81px 0 48px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.lg--search--bar__form,
+.sm--search--bar__form {
```

`--` is used for BEM modifiers, just use single dash for container class names.
```suggestion
.lg-search-bar__form,
.sm-search-bar__form {
```